packages/neon/neon/lib/src/settings/models/select_option.dart

```diff
@@ -12,10 +12,17 @@ class SelectOption<T> extends Option<T> {
     required super.label,
     required super.defaultValue,
     required final Map<T, LabelBuilder> values,
+
+    /// Force loading the stored value.
+    ///
+    /// This is needed when [values] is empty but the stored value should still
+    /// be loaded.
+    /// This only works when [T] is of type String.
+    final bool forceLoadValue = false,
```

Comment:
Can't we just always force load? I fear that future devs will forget to set this option manually

Reply:
What about keeping forceLoadValue and defaulting to true?

Reply:
Is the false value useful for anything if it is broken without it?

Reply:
It's not strictly broken.
The problem only occurs when `values` is empty when initializing.

Force loading will only work when the generic is of type `String` as we wouldn't know how to desetialize it without values being set.

It just so happens that we mostly use Strings (the ID of something) and use late loading quite some times

Reply:
Ah, makes sense. How about only defaulting to true if the type is String?